wnfs/src/common/link.rs

```diff
@@ -48,7 +48,7 @@ impl<T> Link<T> {
             Self::Encoded { cid, .. } => Ok(cid),
             Self::Decoded { value, cid_cache } => {
                 cid_cache
-                    .get_or_try_init(async { store.put_async_serializable(value).await })
```

Comment:
Yeah weird I swear we fixed these in some other PR recently.
Hopefully nothing went wrong in some merge commit previously :eyes:

Reply:
Ah nevermind. This is fixed in `main`, the PR just needs to be rebased :)

Reply:
ill rebase

Reply:
ok checkem